hs_core/templates/pages/genericresource.html

```diff
@@ -2359,6 +2374,39 @@ <h3 style="color: red" id="file-type-error"></h3>
             }
         });
 
+        var resource_id = $("#resource_id").attr("value");
+
+        // for tracking when a user is selected in add creator/contributor model form
+        $(".deck").bind("DOMNodeInserted",function(){
```

Comment:
No idea how to resolve this. The insert/remove is handled by the autocomplete module (I have no idea how this module works). I am not sure how I change the form element attributes without doing it in javascript.  @ironfroggy can you provide some more details to help me resolve this?